Standards/scs-XXXX-w1-Availability-Zones-Standard.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "SCS Availability Zone Standard: Implementation and Testing Notes"
+type: Supplement
+track: IaaS
+status: Draft
+supplements:
+  - scs-XXXX-vN-Availability-Zones-Standard.md
+---
+
+## Automated Tests
+
+The SCS will also allow small deployments and edge deployments, that both will not meet the requirement for bein divided into multiple Availability Zones.
```

Comment:
```suggestion
The SCS will also allow small deployments and edge deployments, that both will not meet the requirement for being divided into multiple Availability Zones.
```

As mentioned before, it makes no sense to say "The SCS will also allow". The "Sovereign Cloud Stack" is a piece of technology and cannot mandate or regulate anything. I suppose what you want to say is "The standard does not preclude small deployments and edge deployments"